newrelic/provider_test.go

```diff
@@ -113,3 +114,32 @@ func testAccCreateApplication(t *testing.T) {
 
 // 	testAccCleanupComplete = true
 // }
+
+// testCheckResourceAttr validates the value in state for the given name/key combination.
+func testCheckResourceAttr(is *terraform.InstanceState, name string, key string, value string) error {
```

Comment:
Why was this necessary to copy in?  [`TestCheckResourceAttr`](https://github.com/hashicorp/terraform/blob/master/helper/resource/testing.go#L955) is exported by the testing package, does it not get the job done in this case?

Reply:
Well I could've written my own version of this, but this seemed like something we could use in other custom import check functions.

Reply:
I can try to see if I can call one of the functions that TF exports inside the custom check I created. I tried several ways to do that, but couldn't get it to work like we'd want.

Reply:
Or I could do this
```go
func testAccNewRelicAlertPolicyImportStateCheckFunc(resourceName string, t *testing.T) resource.ImportStateCheckFunc {
	return func(state []*terraform.InstanceState) error {
		expectedChannelsCount := "1"
		channelsCount := state[0].Attributes["channel_ids.#"]

		if channelsCount != expectedChannelsCount {
			return fmt.Errorf(
				"%s: Attribute '%s' expected %#v, got %#v",
				resourceName,
				"channel_ids.#",
				expectedChannelsCount,
				channelsCount,
			)
		}

		return nil
	}
}
```